app/code/Magento/Widget/Api/DeleteWidgetInstanceByIdInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Widget\Api;
+
+/**
+ * Command to delete a widget instance by specified widget instance ID
+ * @api
+ */
+interface DeleteWidgetInstanceByIdInterface
```

Comment:
I have doubts regarding a necessity to use an API interface in this particular case. It might be useful if we create an additional web API endpoint or we need to define a strategy in an interface for multiple implementations. But it's not the case. 
The main downside of using an interface here - it does not make much sense. In the future, the interface restricts us in possible architectural changes (if the implementation needs to be changed). So if we declare an interface we are responsible for its support. On the other hand, if we don't have an interface we can change the implementation in more flexible ways.
So the main idea is to use API interfaces only in cases when they are useful but not harmful.

Reply:
Got it! I'll adjust it. Thank you!